var/spack/repos/builtin/packages/py-brian2/package.py

```diff
@@ -0,0 +1,50 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class PyBrian2(PythonPackage):
+    """A clock-driven simulator for spiking neural networks"""
+
+    homepage = "http://www.briansimulator.org"
+    url      = "https://pypi.io/packages/source/B/Brian2/Brian2-2.0.1.tar.gz"
+
+    version('2.0.1', 'df5990e9a71f7344887bc02f54dfd0f0')
+    version('2.0rc3', '3100c5e4eb9eb83a06ff0413a7d43152')
+
+    variant('docs', default=False)
+
+    # depends on py-setuptools@6: for windows, if spack targets windows,
+    # this will need to be added here
+    depends_on('py-setuptools',     type='build')
```

Comment:
You're missing a build dependency:
```python
depends_on('py-cython@0.18:', type='build')
```
Without this, the executable will be significantly slower. You might also have to pass in the `--with-cython` option to `build_args(self)`.